Adding-new-translations-for-i18n.md

```diff
@@ -33,12 +52,138 @@ In the example above, if there is only one notification, then we should change "
 In this example, the # symbol will be replaced by the value of the `notification_number` variable. For a more elaborate tutorial, please refer to the [angular-translate guide for pluralization](http://angular-translate.github.io/docs/#/guide/14_pluralization).
 
 
-## Manually adding translations
+## How to translate lessons and other dynamic text
+
+If you'd like to help translate Oppia's lessons, please get in touch via our [volunteer form](https://forms.gle/BK99fdqBShY7BPKC8). Successful applicants will be invited to join one of Oppia's translation teams.
+
+We really appreciate help with translations to make the lessons accessible for students whose first language isn't English. Thank you for helping out!
+
+
+## How to create an i18n-compliant PR
+
+When developing learner-facing functionality, you must use `I18N_...` strings as placeholders for text content. This enables such strings to be translated. You can see the translations in the [i18n directory](https://github.com/oppia/oppia/tree/develop/assets/i18n), which map translation keys like `I18N_MODULE_STRING_NAME` to the translated strings. When a page is loaded, angular-translate traverses the page's html code and changes the translation key to the appropriate translated string.
+
+### Good i18n practices
+
+Please consider i18n while developing. Not all languages are the same: words have different lengths, pluralization rules differ, sentences have different structures and the direction of writing can be from right to left. As a result, sometimes development practices that are generally good (like code reuse) turn out to be less than ideal for i18n. Here are some important points to take into account:
+- Do not include raw strings, always use a translation key. Even if it is just an exclamation mark (!)
+- When designing a page, plan for the case where strings are twice the length in other languages. Also, think about how the page would look like if the language is written from right to left.
+- Try to include as much text as you can in a single key, so that the translator can provide a more coherent translation. Do not divide a paragraph into multiple keys unless you cannot avoid it. Don't split strings up and concatenate them, since different languages will use a different grammatical order.
+- If you need to include html in your string, such as `<a></a>` tags, try and pass the code as an argument to the translation service.
```

Comment:
how do we do this? an example might help

Reply:
This was mentioned above, so I added a reference link.